js/sezzle/sezzlepay/sezzle_sezzlepay.js

```diff
@@ -10,5 +10,28 @@
     script.type = 'text/javascript';
     script.src = 'https://widget.sezzle.com/v1/javascript/price-widget?uuid=' + merchant_id;
     document.head.appendChild(script);
+    document.sezzleConfig = {
```

Comment:
Same as with WooCommerce, default config should be defined before the script is created. Please move the config to start on line 8.